lib/JSON/Validator/OpenAPI/Mojolicious.pm

```diff
@@ -0,0 +1,72 @@
+package JSON::Validator::OpenAPI::Mojolicious;
+
+use Carp qw(confess);
+use Mojo::Base 'JSON::Validator::OpenAPI';
+
+### request ###
+
+sub _get_request_data {
+  my ($self, $c, $in) = @_;
+
+  return $c->req->url->query->to_hash(1)  if $in eq 'query';
+  return $c->match->stack->[-1]           if $in eq 'path';
+  return $c->req->body_params->to_hash(1) if $in eq 'formData';
+  return $c->req->headers->to_hash(1)     if $in eq 'header';
```

Comment:
I find it important to verify all occurrences of parameter keys. My suggested API shall return all the values as hashes with arrayrefs for multiple values. In Dancer2 I used `Hash::MultiValue->to_mixed_hashref`. `Mojo::Headers` implements a flag argument to produce "mixed hashrefs" in [`$headers->to_hash(1)`](https://metacpan.org/pod/Mojo::Headers#to_hash). I don't know how this is supposed to work for query or body params but I think we should have mixed hashrefs there, too.